dist/docker/redhat/Dockerfile

```diff
@@ -4,10 +4,6 @@ MAINTAINER Avi Kivity <avi@cloudius-systems.com>
 
 ENV container docker
 
-# The SCYLLA_REPO_URL argument specifies the URL to the RPM repository this Docker image uses to install Scylla. The default value is the Scylla's unstable RPM repository, which contains the daily build.
-ARG SCYLLA_REPO_URL=http://downloads.scylladb.com/rpm/unstable/centos/master/latest/scylla.repo
-ARG VERSION=666.development
```

Comment:
You cannot just drop `VERSION`. We added it on purpose in commit de19496ff7a1aeeb22096006ef2309270e9255e2 to ensure that the Jenkins pipelines can make sure the Docker image has the exact same version we're promoting.

Reply:
thank you for highlighting this! I turned back VERSION parameter with default value = '' so that both default and specific version will work.